orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java

```diff
@@ -255,22 +265,31 @@ private static GKInstance createInfEntitySet(GKInstance entitySetInst, boolean o
 		if (inferredEntitySetIdenticals.get(entitySetInst) == null)
 		{
 			// Equivalent to infer_members function in infer_events.pl
-			HashSet<String> existingMemberInstances = new HashSet<String>();
-			ArrayList<GKInstance> membersList = new ArrayList<GKInstance>();
-			logger.info("\tInferring EntitySet member instances...");
-			for (GKInstance memberInst : (Collection<GKInstance>) entitySetInst.getAttributeValuesList(hasMember))
+			HashSet<String> existingMemberInstances = new HashSet<>();
+			ArrayList<GKInstance> infMembersList = new ArrayList<>();
```

Comment:
These two declarations could be "Set" and "List", respectively.

Reply:
The general term is "programming to an interface (instead of implementation)": https://stackoverflow.com/questions/1992384/program-to-an-interface-what-does-it-mean